src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -2,56 +2,68 @@
+| ParallelAcquisitionTechnique   | RECOMMENDED           |               | The type of parallel imaging used (e.g. GRAPPA, SENSE). Corresponds to DICOM Tag 0018, 9078 `Parallel Acquisition Technique`.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             |
+| PartialFourier                 | RECOMMENDED           |               | The fraction of partial Fourier information collected. Corresponds to DICOM Tag 0018, 9081 `Partial Fourier`.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             |
+| PartialFourierDirection        | RECOMMENDED           |               | The direction where only partial Fourier information was collected. Corresponds to DICOM Tag 0018, 9036 `Partial Fourier Direction`.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
```

Comment:
what datatype is this?

Reply:
This is unused, except strings containing a space (`" "`). It reads like a string, though.